src/modules/redesign/i18n/it_IT/verband/protokoll.json

```diff
@@ -0,0 +1,19 @@
+{
+	"affected": "Interessato",
+	"amount": "{n} voce filtrata | {n} voci filtrate",
+	"description": "descrizione",
+	"executor": "trigger",
```

Comment:
non so cosa sia executor in questo caso, ma presumo sia stato dimenticato

Reply:
ho scritto azione, sul momento non sapevo cosa scrivere al posto di trigger